docs/how-to/make-plots.md

```diff
@@ -1,6 +1,5 @@
 ### Making plot using the outputs from `integrate`
-Since the outputs from `x.integrate` are matrices, you can use all normal MATLAB
-plotting functionality.
+Since the outputs from `x.integrate` are matrices when `x.output_structure = false`, you can use all normal MATLAB plotting functionality.
```

Comment:
it's not output_structure, but `output_type`

it can be 0 (default), 1 or 2

0 -- matrices returned in seperate variables
1 -- structure
2 -- structure + spikes detected in C++ code